src/dotnet/CarbonAware.Aggregator/src/CarbonAware.Aggregator.csproj

```diff
@@ -0,0 +1,14 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
Are we calling these aggregators or calculators?

Reply:
I honestly wasn't sure where we landed on that. @juzuluag thoughts?

Reply:
aggregators is a OOD term. Calculator is more narrowed to the this business logic.
So I'm more inclined to OOD, since an aggregator could be anything (i.e calculator, ...)